### PR TITLE
[upload_symbols_to_crashlytics] Fix `gsp_path` parameter will not expand when use with `api_token` parameter.

### PR DESCRIPTION
**Pull request by r-plus**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

I passed `gsp_path` and `api_token` parameter to upload_symbols_to_crashlytics lane. (yes, I know using both is not necessary. But i think should work even if using both parameter.)

#### Situation

- pass to zip file to upload dSYM
- pass `gsp_path` parameter that is relative path.
- pass `api_token` parameter.

In this case, upload_symbols_to_crashlytics lane behavior will be

1. `gsp_path` parameter will not expand
2. current directory will change to temporary directory for unzip
3. upload_dsym command will use with `-gsp` argument that is not expanded path of `gsp_path`
4. gsp configuration file is not found error will occurred like this.

```
INFO [2020-05-19 05:14:54.13]: -------------------------------------------
INFO [2020-05-19 05:14:54.13]: --- Step: upload_symbols_to_crashlytics ---
INFO [2020-05-19 05:14:54.13]: -------------------------------------------
INFO [2020-05-19 05:14:54.13]: Extracting '/Users/distiller/App/App.app.dSYM.zip'...
INFO [2020-05-19 05:14:54.13]: $ unzip -qo /Users/distiller/App/App.app.dSYM.zip
INFO [2020-05-19 05:14:57.28]: Uploading 'EB65ACFF-2249-3751-80DD-21AE4E27D786.dSYM'...
DEBUG [2020-05-19 05:14:57.28]: upload_dsym using command: /Users/distiller/App/Pods/FirebaseCrashlytics/upload-symbols -gsp Configurations/Firebase/Debug/GoogleService-Info.plist -p ios /private/var/folders/bq/mjrgbpkx5h1g_b22fpv0tlzc0000gn/T/d20200519-17327-1v727ia/EB65ACFF-2249-3751-80DD-21AE4E27D786.dSYM
ERROR [2020-05-19 05:14:57.33]: Shell command exited with exit status 13 instead of 0.
...
```

### Description
This PR change to expand `gsp_path` parameter even if exist `api_token` parameter to works fine.

### CODE REVIEW

**Comment by r-plus** [comment]
@googlebot I signed it!

File: fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb [hunk 1]
```diff
@@ -119,7 +119,7 @@ def self.find_api_token(params)
       end
 
       def self.find_gsp_path(params)
-        return if params[:api_token]
+        return if params[:api_token] && !params[:gsp_path]
```
Comment (minuscorp) [inline fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb:122]: Readability?
```suggestion
        return if params[:api_token] && params[:gsp_path].nil?
```

Comment (r-plus) [reply fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb]: thx :)

**Review by minuscorp — APPROVED** [review]
LGTM 🚀

**Review by snatchev — APPROVED** [review]
LGTM

**Pull request merged**